routes/admin_stock/index.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+const { setItemStockFromDB } = require('./util')
+
+const schema = {
+  description: 'DynamoDB admin functions. Change stock of item in a machine',
+  tags: ['routes'],
+  summary: 'DynamoDB admin functions',
+  body: {
+    type: 'object',
+    required: ['machine_id', 'item_id', 'item_stock'],
+    properties: {
+      machine_id: { type: 'string' },
+      item_id: { type: 'string' },
+      item_stock: { type: 'object' }

```

Comment:
I told you to make this an object assuming this path would be for multiple items at a time. Since it is for just one item at a time, your original item_id:string and item_stock:number makes more sense